src/adiimu_ros_publisher.cpp

```diff
@@ -72,6 +72,8 @@ void AdiImuRosPublisher::run()
                     m_message.delta_angle.x, m_message.delta_angle.y, m_message.delta_angle.z);
         RCLCPP_INFO(rclcpp::get_logger("rclcpp_adiimu"), "Publishing adi imu temperature data: '%f' ",
                     m_message.temp);
+        RCLCPP_INFO(rclcpp::get_logger("rclcpp_adiimu"), "Publishing adi imu count: '%d' ",
+                    m_message.count);
```

Comment:
you add this here and then you remove it in another later commit... just don't add it here at all